src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return (a) => {
+    return (b) => {
+      count++;
+
+      if (count > 3 && count % 2 === 0) {
```

Comment:
Here, too, you can improve readability and put these calculations in a separate variable, for example, `isBroke`